src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -316,6 +316,9 @@
     <ReferenceFromRuntime Include="System.Private.Interop" />
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
     <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" />
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
+      <Aliases>System_Security_Principal</Aliases>
```

Comment:
Path.GetRandomFileName in Path.Uap.cs uses Windows​.Security​.Cryptography.CryptographicBuffer.  The "Windows" assembly requires mscorlib, otherwise you'll get this error: "The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'mscorlib'". System.Private.Interop is needed for the Marshal class.  The rest may be merged.